api/instructor/instructorModel.js

```diff
@@ -1,14 +1,9 @@
 const db = require('../../data/db-config');
 
-const findInstructorCourses = async (profile_id) => {
-  return db('instructors')
-    .leftJoin(
-      'courses',
-      'instructors.instructor_id',
-      '=',
-      'courses.instructor_id'
-    )
-    .where('instructors.profile_id', profile_id);
+const findInstructorCourses = async (instructor_id) => {
+  return db('courses as c')
+    .leftJoin('instructors as i', 'i.instructor_id', 'c.course_id')
```

Comment:
This is a good catch to get rid of the "=" and labeling the 2 tables as "i" and "c".